### PR TITLE
Home all levels

### PR DESCRIPTION
**Pull request by greenEkatherine**
Part of #185

### CODE REVIEW

File: app/src/main/res/raw/data.json [hunk 1]
```diff
@@ -7806,198 +7806,1434 @@
...
-    ],    
-	[
-    {"EN":{
-        "word": "kitchen",
-        "topics": ["home", "cooking and eating"],
-        "level": "beginner(A1)"
+    ],
+    [
+        {"EN":{
+            "word": "borrow",
```
Comment (omen273) [inline app/src/main/res/raw/data.json:7977]: I think it is not "арендовать"

File: app/src/main/res/raw/data.json [hunk 2]
```diff
@@ -7806,198 +7806,1434 @@
...
+        {"EN":{
+            "word": "broom",
+            "topics": ["home"],
+            "level": "advanced(C1)"
         }},
-    {"RU": {
-        "word": "ключ",
-        "topics": ["дом"]
+        {"RU": {
+            "word": "метла",
```
Comment (omen273) [inline app/src/main/res/raw/data.json:7993]: веник

File: app/src/main/res/raw/data.json [hunk 3]
```diff
@@ -7806,198 +7806,1434 @@
...
+    ],
+    [
+        {"EN":{
+            "word": "couch",
+            "topics": ["home"],
+            "level": "upper-intermediate(B2)"
+        }},
+        {"RU": {
+            "word": "кушетка",
+            "topics": ["дом"]
```
Comment (omen273) [inline app/src/main/res/raw/data.json:8170]: диван

File: app/src/main/res/raw/data.json [hunk 4]
```diff
@@ -7806,198 +7806,1434 @@
...
+        }}
+    ],
+    [
+        {"EN":{
+            "word": "plug",
+            "topics": ["home"],
+            "level": "advanced(C1)"
+        }},
+        {"RU": {
+            "word": "штепсельная вилка",
```
Comment (omen273) [inline app/src/main/res/raw/data.json:8859]: Maybe simply "вилка"

**Review by omen273 — CHANGES_REQUESTED** [review]
Please address my comments

**Comment by omen273** [comment]
Please add sink also

File: app/src/main/res/raw/data.json (hunk 4 above)
Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:8859]: вилка в контексте дома может означать столовый прибор

**Comment by greenEkatherine** [comment]
> Please add sink also

we already have it

File: app/src/main/res/raw/data.json (hunk 3 above)
Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:8159]: done

Comment (greenEkatherine) [reply app/src/main/res/raw/data.json:8159]: done

File: app/src/main/res/raw/data.json (hunk 1 above)
Comment (greenEkatherine) [reply app/src/main/res/raw/data.json]: removed

**Review by omen273 — APPROVED** [review]

**Pull request merged**